### PR TITLE
Identify slow CLI tests

### PR DESCRIPTION
**Pull request by manno**
Set slowSpecThreshold to 200s

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -80,7 +80,7 @@ test-acceptance-api: showfocus embed_files
 	ginkgo -nodes ${GINKGO_NODES} -stream -randomizeAllSpecs --flakeAttempts=${FLAKE_ATTEMPTS} -failOnPending acceptance/api/v1/.
 
 test-acceptance-cli: showfocus embed_files
-	ginkgo -nodes ${GINKGO_NODES} -stream -randomizeAllSpecs --flakeAttempts=${FLAKE_ATTEMPTS} -failOnPending acceptance/.
+	ginkgo -nodes ${GINKGO_NODES} -v -slowSpecThreshold 200 -randomizeAllSpecs --flakeAttempts=${FLAKE_ATTEMPTS} -failOnPending acceptance/.
```
Comment (jimmykarily) [inline Makefile:83]: why not the other suites too?

Comment (andreas-kupries) [reply Makefile]: See commit d14244256dfedb53076714c66cdde9efe4e2741d

**Review by andreas-kupries — APPROVED** [review]
It looks ok.

**Review by andreas-kupries — APPROVED** [review]
Still looks ok, with the change from `-v` to `-stream`.

**Review by jimmykarily — APPROVED** [review]

**Pull request merged**